packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -41,6 +56,18 @@ export class RedirectsMiddleware {
     return this.handler;
   }
 
+  protected excludeRoute(pathname: string) {
```

Comment:
Please ensure that built-in excludeRoute method and the excludeRoute function passed via RedirectsMiddlewareConfig are called in middleware handler